### PR TITLE
Update/gh pages deploy

### PR DESCRIPTION
**Pull request by geealbers**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [ ] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

no

### Please describe the goal of this pull request and the changes that were made.

After helping @shineslike earlier with deploying to GitHub Pages, I had a few ideas to make the process a little smoother. There are more steps now, but it avoids making junky deploy commits into the project's `main` branch, and also deletes the existing deploy ahead of time in order to avoid an error I was sometimes getting.

Finally, I tested this on a new repo and given success there and on Suse's project, I've removed the warning we had in place.

### CODE REVIEW

**Comment by Erin-Cecele** [comment]
@geealbers, it looks like you only really added one or two new steps and essentially clarified the others. It reads well. I only noted one incomplete thought/sentence. 

Giving this section a look with fresh eyes, it feels beneficial to add a pros and cons intro to both Netlify and GH Pages. GH Pages' instructions look so complicated. If I were a non-technical person looking at this for the first time, I would immediately choose Netlify. I think it would be helpful to explain the difference between the two options better. Maybe we can take a moment to jot down some bullet points together, and I can add them to this branch.

Also, Maybe this isn't necessary, but I'd love to test this out. What's the best way to go about that?

**Comment by geealbers** [comment]
@Erin-Cecele you mentioned you "noted one incomplete thought/sentence" but I don't see that noted anywhere?

Would be great if you could test it. The best way would be to start a new project, publish it to your personal GitHub, and then walk through the steps to deploy it there.

Also, good idea to have some more explicit comparison between GitHub and Netlify as options. I'd like to get this PR merged so we can share it with Suse sooner than later, so let's put the comparison in with a new PR. But here are some initial thoughts:

- GitHub pages requires a multi-step command-line process for manual deploy, or (until we can offer more documentation) a developer to set up continuous deployment with GitHub Actions
- Netlify manual deploy is a relatively simple drag and drop, and it's continuous deploy can be set up without a developer given our documentation of it
- Netlify requires maintaining another account, on top of GitHub, which some people may be reticent about, but you can log in with GitHub so that's probably trivial

I think the final factor would be pricing. So for our comparison, we'd probably want to outline what each service offers at a Free level, and what the rough cost is for the first tier up.

**Review by Erin-Cecele — COMMENTED** [review]
Ack sorry I didn't submit review!

File: content/docs-v1/site-deploy.md [hunk 1]
```diff
@@ -93,28 +93,30 @@ Should you need to make any updates to your site, just merge the changes and Net
...
 
-4. In the `_site` directory, add an empty file called `.nojekyll`, which lets GitHub know the site you're publsihing [isn't a Jekyll site](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/).
+7. If you have previously deployed the project to GitHub Pages, delete that old version by running `git push origin --delete gh-pages`. If this is your first deploy, skip this step.
 
-5. Next run `git add -f _site && git commit -m "Github pages deploy"` this will add the contents of the `_site` directory to your project's git history so that it can be pushed to GitHub.
+8. Run `git subtree push --prefix _site origin gh-pages` to push the committed `_site` files to a new `gh-pages` branch of your repo on GitHub.
 
-6. Run `git subtree push --prefix _site origin gh-pages` to push the committed `_site` files to a new `gh-pages` branch of your repo on GitHub.
+9. In your command-line shell, run `git checkout main && git branch -D gh-deploy` to delete the temporary deploy branch and to switch back to your `main` branch. If you were publishing from
```
[the remark this answers is not in the document]
Comment (geealbers) [reply content/docs-v1/site-deploy.md]: lol, thanks!

**Comment by geealbers** [comment]
Merging in now, but we'll revisit to add the GitHub/Netlify comparison and/or if any issues arise in @Erin-Cecele's testing

**Pull request merged**